releasenotes/notes/fix-course-export-b35c78a22be0c6b5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Markdown content will now be exported between opening and
+    closing <markdown></markdown> instead of an attribute of <markdown/>.
```

Comment:
Missing "tags" before "instead", and I'd reword this to "... instead of the markdown element's "data" attribute."